rilldata/rill
# Cloud UI: Add a "Share" button to dashboards

## Checklist

## Summary
#### Issue addressed: 
[Notion PRD](https://www.notion.so/rilldata/Sharing-in-Rill-cloud-78d8a04ea24148059e7295173db7476e)

#### Details:
This PR:
- Adds a "Share" secondary button to Cloud dashboards
- Moves the "Ask for help" CTA into the Avatar Menu
- Tweaks styles in the Avatar Menu to match mocks & ensure consistency w/ the Share CTA

One implementation note: for the Share CTA, I've used a Popover component from `svelte-headless-ui`. It'd be more appropriate to use a [Menu](https://svelte-headlessui.goss.io/docs/2.0/menu) component from `svelte-headless-ui`, but we haven't yet styled one to match our existing Menus.

## Steps to Verify
1. Go to a dashboard in the Cloud UI
2. Interact with the dashboard to add some state to the URL
3. Click "Share", copy the link, open the link in a new browser
4. Ensure the state is retained
